core/metacling/src/TClingCallbacks.cxx

```diff
@@ -158,13 +164,18 @@ class AutoloadLibraryMU : public llvm::orc::MaterializationUnit {
 };
 
 class AutoloadLibraryGenerator : public llvm::orc::DefinitionGenerator {
+   const TClingCallbacks &CB;
```

Comment:
Can you please put the data member next to the others, and rename it as `fCallbacks` to be consistent?